modules/installation-vsphere-infrastructure.adoc

```diff
@@ -73,17 +71,17 @@ Installing a cluster on VMware vSphere versions 7.0 and 7.0 Update 1 is deprecat
 |Component | Minimum supported versions |Description
 
 |Hypervisor
-|vSphere 7.0 Update 2 and later with virtual hardware version 15
+|vSphere 7.0 Update 2 (or later) or vSphere 8.0 Update 1 (or later) with virtual hardware version 15
```

Comment:
This is not enough to hold up a merge review but in this instance you put or later in parentheses and in the rest your do not. Maybe you could fix it in a separate PR if you wanted @dfitzmau